Library/Formula/youtube-dl.rb

```diff
@@ -5,8 +5,8 @@
 # https://pypi.python.org/pypi/youtube_dl
 class YoutubeDl < Formula
   homepage "http://rg3.github.io/youtube-dl/"
```

Comment:
This line needs changing to ` homepage "https://rg3.github.io/youtube-dl/" `

Otherwise the PR is great, Thanks! :)